addons/packages/vsphere-cpi/1.20.0/bundle/config/schema.yaml

```diff
@@ -0,0 +1,79 @@
+#! schema.yml
+
+#@data/values-schema
+#@schema/desc "OpenAPIv3 Schema for vsphere-cpi"
+---
+vsphereCPI:
+  #@schema/desc "The cryptographic thumbprint of the vSphere endpoint's certificate"
+  tlsThumbprint: ""
+  #@schema/desc "The IP address or FQDN of the vSphere endpoint"
+  #@schema/nullable
+  server: 10.1.1.1
+  #@schema/desc "The datacenter in which VMs are created/located"
+  #@schema/nullable
+  datacenter: ""
+  #@schema/desc "Username used to access a vSphere endpoint"
+  #@schema/nullable
+  username: ""
+  #@schema/desc "Password used to access a vSphere endpoint"
+  #@schema/nullable
+  password: ""
+  #@schema/desc "The region used by vSphere multi-AZ feature"
+  #@schema/nullable
+  region: ""
+  #@schema/desc "The zone used by vSphere multi-AZ feature"
+  #@schema/nullable
+  zone: ""
+  #@schema/desc "The flag that disables TLS peer verification"
+  insecureFlag: False
+  #@schema/desc "IP family"
```

Comment:
Could you change the description to "The IP family vsphere cpi should be configured with" , to be consistent with ipFamily description in => https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/calico/3.19.1/bundle/config/schema.yaml#L10